Core/COEditingContext.m

```diff
@@ -369,6 +392,11 @@ - (void)undeletePersistentRoot: (COPersistentRoot *)aPersistentRoot
  */
 - (void)updateDeadRelationshipCacheForUndeletedPersistentRoot: (COPersistentRoot *)aPersistentRoot
 {
+	// FIXME:
```

Comment:
I ended up removing this code entirely, since it's not needed at all. I wrote some tests to ensure the dead relationship cache is still properly cleaned. See my last commit in this pull request.